consensus-types/blocks/roblob.go

```diff
@@ -0,0 +1,42 @@
+package blocks
+
+import (
+	"github.com/pkg/errors"
+	ethpb "github.com/prysmaticlabs/prysm/v4/proto/prysm/v1alpha1"
+)
+
+var errNilBlockHeader = errors.New("received nil beacon block header")
```

Comment:
Feels like this and the nil block error should probably be exported so we can check them from callers

Reply:
Do we need to check them external to the package? Sometimes we have unexported errors like this so that the package's unit test can check the error, but external users don't need access to the error type.